### PR TITLE
Styled letters pages for Amp

### PR DESCRIPTION
**Pull request by zeftilldeath**
Added styling for loads of article types:

analysis, comment, feature, review, editorial, live, dead, special-report, letters

screen shot 2015-10-01 at 10 41 43
etc

### CODE REVIEW

**Comment by sndrs** [comment]
:+1:

File: common/app/views/fragments/amp/stylesheets/tones.scala.html [hunk 1]
```diff
@@ -21,7 +21,8 @@
     color: #fff;
 }
 
-.tonal--tone-comment .tonal__standfirst {
+.tonal--tone-comment .tonal__standfirst,
```
Comment (stephanfowler) [inline common/app/views/fragments/amp/stylesheets/tones.scala.html]: The AMP spec mandates non-nested selectors, ie strict BEM. Not sure that they actually throw validation errors for it though. Have you tried validating it?

Comment (zeftilldeath) [inline common/app/views/fragments/amp/stylesheets/tones.scala.html]: Is there a tool I can use for that?

Comment (stephanfowler) [inline common/app/views/fragments/amp/stylesheets/tones.scala.html]: Add `#development=1` to the url. Sorry shoulda said that.

Comment (stephanfowler) [inline common/app/views/fragments/amp/stylesheets/tones.scala.html]: PS the spec says:

Allowed selectors are of the form:
`.class`
`#id`
`tag-name`
`selector, selector`
`media queries`

Comment (zeftilldeath) [inline common/app/views/fragments/amp/stylesheets/tones.scala.html]: Ah right, I'm confused now, as all of the colour/tonal specific css is through nested selectors. Shall we have a chat when your back upstairs?

Comment (stephanfowler) [inline common/app/views/fragments/amp/stylesheets/tones.scala.html]: Hmm. We need clarity on whether that rule is actually enforced or not. Will ask Google, but carry on as you are for now.

Comment (stephanfowler) [inline common/app/views/fragments/amp/stylesheets/tones.scala.html]: OK, Google says they're not enforcing this for the Preview. So nested selectors are OK - for now.

File: common/app/views/fragments/amp/stylesheets/tones.scala.html [hunk 2]
```diff
@@ -113,8 +135,19 @@
...
+.tonal--tone-review .tonal__standfirst ul>li:before,
+.tonal--tone-review .tonal__standfirst .bullet:before,
+.tonal--tone-editorial .tonal__standfirst ul>li:before,
+.tonal--tone-editorial .tonal__standfirst .bullet:before,
+.tonal--tone-live .tonal__standfirst ul>li:before,
+.tonal--tone-live .tonal__standfirst .bullet:before,
+.tonal--tone-special-report .tonal__standfirst ul>li:before,
+.tonal--tone-special-report .tonal__standfirst .bullet:before,
+.tonal--tone-letters .tonal__standfirst ul>li:before,
+.tonal--tone-letters .tonal__standfirst .bullet:before {
```
Comment (sammorrisdesign) [inline common/app/views/fragments/amp/stylesheets/tones.scala.html]: Gross

**Comment by sammorrisdesign** [comment]
:+1:

File: common/app/views/fragments/amp/stylesheets/tones.scala.html (hunk 2 above)
Comment (zeftilldeath) [inline common/app/views/fragments/amp/stylesheets/tones.scala.html]: Haha yep. We definitely need to look at a more streamlined way of doing this.

**Pull request merged**